src/cli.js

```diff
@@ -12,6 +12,7 @@ program
   .option("-w, --watch", "Watch tests")
   .option("-v, --verbose", "Verbose tests")
   .option("-f, --fun", "Fun tests")
+  .option("-p, --port [port]", "Port number to launch test RPC on")
```

Comment:
I think just `"-p, --port"` here

Reply:
The `[port]` needs to be there to accept the passed in port number